tests/specification_tests/test_capabilities.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+from enum import Enum
+import z3
+from scipy.stats import norm, kstest
+
+from causal_testing.specification.capabilities import Capability, TreatmentSequence
+
+
+class TestCapability(unittest.TestCase):
+
+    """
+    Test the Capability class for basic methods.
+    """
+
+    def setUp(self) -> None:
+        pass
+
+    def test_repr(self):
+        cap = Capability("v", 1, 0, 1)
+        self.assertEqual(str(cap), "(v, 1, 0-1)")
```

Comment:
@jmafoster1 Should this be `"(v, 1, 0, 1)"` instead of `"(v, 1, 0-1)"`?

Reply:
I thought the `0-1` notation for stringing capabilities would be easier to read, then you know that the last element represents the duration of the capability. Otherwise, I'd be constantly forgetting what the last three elements represented!